test/functional/wallet_basic.py

```diff
@@ -642,6 +642,12 @@ def run_test(self):
         assert not address_info["iswatchonly"]
         assert not address_info["isscript"]
         assert not address_info["ischange"]
+        assert_equal(address_info['use_txids'], [])
+
+        # Test getaddressinfo 'txids' field
```

Comment:
94b20a9f9e81e5b0ef45a0259b81be83b602c325

```suggestion
        # Test getaddressinfo 'use_txids' field
```

-----

Could test with multiple txids
```diff
-        # Test getaddressinfo 'txids' field
-        txid = self.nodes[0].sendtoaddress("mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ", 1)
-        address_info = self.nodes[0].getaddressinfo("mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ")
-        assert_equal(address_info['use_txids'], [txid])
+        # Test getaddressinfo 'use_txids' field
+        addr = "mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ"
+        txid_1 = self.nodes[0].sendtoaddress(addr, 1)
+        address_info = self.nodes[0].getaddressinfo(addr)
+        assert_equal(address_info['use_txids'], [txid_1])
+        txid_2 = self.nodes[0].sendtoaddress(addr, 1)
+        address_info = self.nodes[0].getaddressinfo(addr)
+        assert_equal(address_info['use_txids'], [txid_1, txid_2])
```

Reply:
Done